### PR TITLE
Ccg trad clients snapshots #18879

### PR DESCRIPTION
**Pull request by keichwa**
https://github.com/SUSE/spacewalk/issues/18879

Plus drive-by edit for clarity.

# Description

Short summary of why you created this PR (if you added documentation, please add any relevant diagram).

# Target branches

* Which documentation version does this PR apply to?
* Does this content need to be backported? If this is the case, create an issue for tracking it and add the link to this PR.
* Whenever possible, cross-reference each backport PR here, so that all backports can be easily accessed from the description.

- [ ] master
- [ ] manager-4.3

# Links
- This PR tracks issue #<insert spacewalk issue, if any>
- Related development PR #<insert PR link, if any>

### CODE REVIEW

**Review by 0rnela — CHANGES_REQUESTED** [review]
Please add .changelog.

**Review by 0rnela — APPROVED** [review]

File: modules/client-configuration/pages/client-operations.adoc [hunk 1]
```diff
@@ -1,10 +1,8 @@
 [[client.operations]]
 = Client Operations
 
-In addition to registering, upgrading, or deleting clients other operations can be performed.
+In addition to registering, upgrading, installing software, or deleting clients other operations can be performed.
+{productname} clients can be managed individually, or organized in groups using System Set Manager, System Groups, or Configuration Management.
```
Comment (vzhestkov) [inline modules/client-configuration/pages/client-operations.adoc:5]: For me it's mot really clear what does it mean to be `organized in groups using ..., or Configuration Management`? Sounds like Configuration Management can organize systems to the groups. Sounds very confusing. I see the phrase was before the change, but for me it sounds like we need to fix it.

Comment (keichwa) [reply modules/client-configuration/pages/client-operations.adoc]: yes, thanks for feedback.  Please check again, I tried to simplify and enhance it.  This text is meant as an intro to the following sections that will actually explain the features listed here.  If sth is missing or still unclear do not hesitate to add proposals or questions.

Comment (vzhestkov) [reply modules/client-configuration/pages/client-operations.adoc]: Thank you. Now it sounds correct and not causing any confusions. :+1:

**Pull request merged**